shared/contexts/LanguageProvider.tsx

```diff
@@ -1,6 +1,13 @@
 import React, { createContext, useContext, useEffect, useState, PropsWithChildren } from 'react'
 import i18n from 'i18n-js'
 import { useLogger } from '@shared-contexts/NativeLoggingProvider'
+import dayjs from 'dayjs'
+import localizedFormat from 'dayjs/plugin/localizedFormat'
+dayjs.extend(localizedFormat)
+require('dayjs/locale/de')
```

Comment:
Does ES 2015 importing works here? Instead of require

Reply:
yes, works
thanks